src/account/account.controller.ts

```diff
@@ -39,6 +48,35 @@ export class AccountController {
     return this.accountService.getAllAccounts();
   }
 
+  @Get('attributes/stream')
```

Comment:
If you want to be more consistent on the swagger page you can switch the order of the attributes/stream and attributes. All the other endpoints have the stream endpoint after